src/ResourceManager/Cdn/Commands.Cdn/Models/Profile/PSProfileResourceState.cs

```diff
@@ -0,0 +1,26 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+namespace Microsoft.Azure.Commands.Cdn.Models.Profile
+{
+    public enum PSProfileResourceState
```

Comment:
Do you really want this as an enum?  Enums on deserialization are essentially just a timebomb waiting to throw when the enum value changes on the service.  Think carefully about whether your enums are actually guaranteed not to add new values, for those enums used on deserialization.

Enums on serialization are entirely appropriate